### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by kataev**

### CODE REVIEW

File: trafaret_config/error.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def _convert(parent_marks, prefix, err, data):
         marks = (cmarks.get(key) or cmarks.get(str(key)) or
                  cmarks.get('__self__') or parent_marks)
         if isinstance(getattr(suberror, 'error', None), dict):
-            if is_alter:
+            if is_alter or data is None:
```
Comment (tailhook) [inline trafaret_config/error.py]: Do you have a test for this bug?

Comment (kataev) [reply trafaret_config/error.py]: No, I lost reproducing error trafaret scheme :(

Comment (kataev) [reply trafaret_config/error.py]: Oh, just a minute, i found it.

Comment (kataev) [reply trafaret_config/error.py]: Add test and change fix. Please review again.

**Comment by tailhook** [comment]
Merged. Thanks!

**Pull request merged**